content/docs/latest/170266662.md

```diff
@@ -59,35 +59,35 @@ This configuration will be set to SAML to indicate that the server will use SAML
 
 <property>  
   <name>hive.server2.saml2.idp.metadata</name>  
-  <value>path\_to\_idp\_metadata.xml</value>  
+  <value>path_to_idp_metadata.xml</value>  
 </property>
```

Comment:
I guess this property definitions should be inside code snippets cause they don't appear in the page.

Reply:
I can address this in a separate PR.